apps/webhook/src/shared/framework/user.decorator.ts

```diff
@@ -0,0 +1,57 @@
+import { createParamDecorator, UnauthorizedException } from '@nestjs/common';
+import * as jwt from 'jsonwebtoken';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const UserSession = createParamDecorator((data, ctx) => {
```

Comment:
So this is just code I have moved from api to application generic and would not like to do fixes in it right now for this PR, because it is already massive. Not sure I am able to answer your question... @scopsy ?